### PR TITLE
upgrade KinD, Helm, valdcli, telepresence, tensorlfow, operator-sdk, …

### PR DESCRIPTION
**Pull request by rinx**
…helm-docs

### Description:

upgrade tools.

### Related Issue:

nothing

### How Has This Been Tested?:

nothing

### Environment:

- Go Version: 1.15.2
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [X] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by datelier — APPROVED** [review]

File: versions/TENSORFLOW_C_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.15.0
+2.3.0
```
Comment (kpango) [inline versions/TENSORFLOW_C_VERSION:1]: I thought we have a problem with tf v2.0.0~2.2.0, can we use v2.3.0 ? If we can use,it should be v2.3.1

Comment (rinx) [reply versions/TENSORFLOW_C_VERSION:1]: @kpango there's no problem. @datelier confirmed that v2.3.0 is okay.

there's no prebuilt lib of version v2.3.1 here, so it should be v2.3.0.
https://www.tensorflow.org/install/lang_c

Comment (kpango) [reply versions/TENSORFLOW_C_VERSION:1]: okay, I understand LGTM

Comment (rinx) [reply versions/TENSORFLOW_C_VERSION:1]: thanks

**Pull request merged**